### PR TITLE
feat: dump timing data for loader (#2521)

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

Ref https://github.com/eggjs/egg/issues/1898

### CODE REVIEW

File: lib/egg.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ class EggApplication extends EggCore {
     this.ready(() => process.nextTick(() => {
       const dumpStartTime = Date.now();
       this.dumpConfig();
+      this.dumpTiming();
```
Comment (atian25) [inline lib/egg.js]: 本地开发的情况下，重启，会不会导致这个目录爆炸？

Comment (popomore) [reply lib/egg.js]: 为啥会爆炸

Comment (atian25) [reply lib/egg.js]: 每重启一次生成一个新的文件？

Comment (popomore) [reply lib/egg.js]: 开发环境会删的

**Pull request merged**